src/effects/Reverb.cpp

```diff
@@ -183,31 +190,43 @@ bool EffectReverb::ProcessInitialize(
                     mParams.mToneLow,
                     mParams.mToneHigh,
                     BLOCK,
-                    mP[i].wet);
+                    state.mP[i].wet);
    }
 
    return true;
 }
 
 bool EffectReverb::ProcessFinalize()
 {
-   for (unsigned int i = 0; i < mNumChans; i++)
+   return InstanceDeinit(mMaster);
+}
+
+bool EffectReverb::InstanceDeinit(ReverbState& state)
```

Comment:
Why is this function here? -- Because, this file is one of the few places missed by the systematic cleanup of naked news and deletes in favor of smart pointers that I did tyears ago.

Let's fix this with a more modern smart pointer while disturbing this file.

Reply:
I'll be happy to bring this code into the early '10s...